dev-config.ini

```diff
@@ -0,0 +1,7 @@
+[deployment]
+conf_dir = ./cloud-config
```

Comment:
For temporary backwards compatibility the default config repo is set to  the repo's cloud-config.  To override, and use the devops versions,  copy this file to .dev-config.ini and change conf_dir = location-of-devops-cloud-config.

Eventually, the dev-config.ini will default to a demonstration deployment and not our internal aws based cloud deployment.